src/features/user/RegisterTrees/RegisterTrees.tsx

```diff
@@ -29,22 +26,19 @@ import SingleContribution from './RegisterTrees/SingleContribution';
 import materialTheme from '../../../theme/themeStyles';
 import { ErrorHandlingContext } from '../../common/Layout/ErrorHandlingContext';
 
-type overridesNameToClassKey = {
-  [P in keyof MuiPickersOverrides]: keyof MuiPickersOverrides[P];
-};
-declare module '@material-ui/core/styles/overrides' {
-  export type ComponentNameToClassKey = overridesNameToClassKey;
-}
+import MuiDatePicker from '@mui/lab/MobileDatePicker';
```

Comment:
Color of selected date needs to be adjusted, and font color (when date is selected) should be white.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r866510455

Reply:
Issue created #1486